signalfx-tracing/signalfx-python-tracing/django/blog/blog/secret_settings.py

```diff
@@ -0,0 +1 @@
+SECRET_KEY = '&g^)t(v(wwu!$+=jov@tnfa2*@eq9q0^2mwa&$y1$gr_25ne9c'
```

Comment:
You can define this in settings.py directly.  Having a separate file seems a bit overkill

Reply:
That's strange. I actually didn't want it committed to the rep. It was part of the .gitignore file. It probably got committed when the .gitignore file was moved

Reply:
Fixed